src/telemetry/telemetry.controller.ts

```diff
@@ -90,6 +94,14 @@ export class TelemetryController {
     }
   }
 
+  private getSkippedMeasurements(): string[] {
+    const measurements = this.config.getWithDefault<string>(
+      'SKIP_MEASUREMENTS',
+      '',
+    );
+    return measurements ? measurements.split(',') : [];
```

Comment:
We don't need to check if `measurement` is undefined anymore, right?

Reply:
Not with the `getWithDefault` I don't think

Reply:
But this is there in case it returns `''`. `''.split(',')` will return `[""]` but I wanted it to just return `[]`. 🤷🏻 not that important but slightly more accurate